src/middleware/empty-response.ts

```diff
@@ -0,0 +1,16 @@
+import { Context, Middleware, Next } from 'koa';
+
+export default (): Middleware => (
+  async ({ response }: Context, next: Next): Promise<void> => {
+    await next();
+
+    if (response.body !== undefined) {
+      return;
+    }
+
+    response.body = '';
+    response.remove('Content-Length');
+    response.remove('Content-Type');
+    response.remove('Transfer-Encoding');
```

Comment:
`Content-Language`, `Content-Encoding`, `Content-Disposition` and `Content-Location` too?

Reply:
Or maybe this middleware shouldn't care beyond `Content-Length` and `Content-Type`?